.gitignore

```diff
@@ -0,0 +1,3 @@
+/sites/default/sqlconf.php 
+/interface/main/calendar/modules/PostCalendar/pntemplates/
```

Comment:
These are more for developers, I guess we can add something to the wiki about adding these into the global .gitignore file.